File: src/test/resources/vertx_tests.rb
VertxAssert = Java::IoVertxTesttools::VertxAssert

def init_tests(top)

  VertxAssert.initialize(org.vertx.java.platform.impl.JRubyVerticleFactory.vertx)

  method_name = Vertx.config['methodName']

  puts "method name is #{method_name}"

  self.send(method_name)

end

File: src/test/resources/vertx_tests.py
import vertx
from io.vertx.testtools import VertxAssert
from org.vertx.java.platform.impl import JythonVerticleFactory

def init_tests(locs) :
    VertxAssert.initialize(JythonVerticleFactory.vertx)
    method_name = vertx.config()['methodName']
    locs[method_name]()